src/svea_sensors/params/amcl/localize.yaml

```diff
@@ -1,13 +1,11 @@
-
-
 laser_max_beams: 50 
-odom_model_type: omni-corrected
+odom_model_type: diff-corrected
```

Comment:
How was it again, this improved AMCL a lot? I think I need to double-check it, iirc there was some documented reason for having it `omni-corrected`.

Reply:
Using `omni_corrected`, AMCL will also take into account the `odom_alpha5` parameter, which describes the tendency of the robot to translate (without rotating) perpedincularly to its direction of travel. We think that this cannot be observed given the SVEA kinematics (1,1 platform), unless some lateral drifting happens (e.g. too high travelling speed).